### ISSUE TITLE
'list' object has no attribute 'lower'

### ISSUE THREAD

**Issue by CyberMew** [Diaoul/subliminal#886]
```
Downloading subtitles
Traceback (most recent call last):
File "/usr/bin/subliminal", line 11, in 
load_entry_point('subliminal==2.1.0.dev0', 'console_scripts', 'subliminal')()
File "/usr/lib64/python2.7/site-packages/click/core.py", line 722, in __call__
return self.main(*args, **kwargs)
File "/usr/lib64/python2.7/site-packages/click/core.py", line 697, in main
rv = self.invoke(ctx)
File "/usr/lib64/python2.7/site-packages/click/core.py", line 1066, in invoke
return _process_result(sub_ctx.command.invoke(sub_ctx))
File "/usr/lib64/python2.7/site-packages/click/core.py", line 895, in invoke
return ctx.invoke(self.callback, **ctx.params)
File "/usr/lib64/python2.7/site-packages/click/core.py", line 535, in invoke
return callback(*args, **kwargs)
File "/usr/lib64/python2.7/site-packages/click/decorators.py", line 27, in new_func
return f(get_current_context().obj, *args, **kwargs)
File "/usr/lib64/python2.7/site-packages/subliminal/cli.py", line 400, in download
hearing_impaired=hearing_impaired, only_one=single)
File "/usr/lib64/python2.7/site-packages/subliminal/core.py", line 261, in download_best_subtitles
for s in subtitles], key=operator.itemgetter(1), reverse=True)
File "/usr/lib64/python2.7/site-packages/subliminal/score.py", line 106, in compute_score
matches = subtitle.get_matches(video)
File "/usr/lib64/python2.7/site-packages/subliminal/providers/opensubtitles.py", line 102, in get_matches
matches |= guess_matches(video, guessit(self.movie_release_name, {'type': 'movie'}))
File "/usr/lib64/python2.7/site-packages/subliminal/subtitle.py", line 235, in guess_matches
if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'
```

**Comment by NulledRoot** [comment]
i have a similar error but its whenever i try to submore then one video

`Traceback (most recent call last):
  File "C:/Users/************/Desktop/Python/MovieO/main.py", line 27, in <module>
    subtitles = download_best_subtitles(videos, {Language(lang)})
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\core.py", line 647, in download_best_subtitles
    compute_score=compute_score)
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\core.py", line 211, in download_best_subtitles
    for s in subtitles], key=operator.itemgetter(1), reverse=True)
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\core.py", line 211, in <listcomp>
    for s in subtitles], key=operator.itemgetter(1), reverse=True)
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\providers\opensubtitles.py", line 98, in get_matches
    matches |= guess_matches(video, guessit(self.movie_release_name, {'type': 'movie'}))
  File "C:\Users\************\AppData\Local\Programs\Python\Python36-32\lib\site-packages\subliminal\subtitle.py", line 235, in guess_matches
    if video.format and 'format' in guess and guess['format'].lower() == video.format.lower():
AttributeError: 'list' object has no attribute 'lower'`

**Comment by abescalamis** [comment]
It looks like this issue was fixed, but not merged, look at #889
